### ISSUE TITLE
`// odinfmt enable/disable` with comment inside gets deleted on formatting

### ISSUE THREAD

**Issue by IanLilleyT** [DanielGavin/ols#144]
Before format:
```
package main

main :: proc() {
    //odinfmt: disable
    // comment
    //odinfmt: enable
}
```

After format:
```
package main

main :: proc() {
}
```

**Comment by nico-bb** [comment]
should be fixed with #149

**Comment by DanielGavin** [comment]
Yeah it's fixed.
